UX/ui-service/Common/Tags/design.md

```diff
@@ -0,0 +1,45 @@
+# Tags - Common Component
+* Tags are used in multiple areas throughout the UI.
+* Tags are made up of two parts, a category and value. Some categories allow for more than one value to be associated.  
+* Tags can be applied to a single item or multiple items at one time depending on the use case.
```

Comment:
@serenamarie125 it is this. In some cases (Blueprints) the user is tagging a single item. In other cases, (Services) the user can multi select to tag multiple items at once. I address the multiple values per category later in the PR. Does that make sense?